tasks/Functions-consumer/request.js

```diff
@@ -206,15 +206,14 @@ task("functions-request", "Initiates an on-demand request from a Functions consu
     }
 
     // Listen for fulfillment
-    const requestId = requestTxReceipt.events[2].args.id
     spinner.start(
-      `Request ${requestId} has been initiated. Waiting for fulfillment from the Decentralized Oracle Network...\n`
+      `Request ${requestTxReceipt.events[2].args.id} has been initiated. Waiting for fulfillment from the Decentralized Oracle Network...\n`
```

Comment:
I changed other examples to print a transaction hash here and the request ID only after fulfillment. Maybe do the same here?

Reply:
Is there informational value in having both? something along the lines of:

`RequestId ${reqId} is waiting for fulfillment from the Decentralized Oracle Network.  Track this fulfillment by its Tx hash ${txHash}. `